tests/test_mteec_csc.py

```diff
@@ -40,7 +40,9 @@ def basic_make_csc(self, initial_state, config_dir, simulation_mode, **kwargs):
 
     async def test_standard_state_transitions(self):
         async with self.make_csc(
-            initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1
+            initial_state=salobj.State.STANDBY,
+            config_dir="tests/data/config",
```

Comment:
At some point please update this to a CWD-independent value. I usually use this:
```
CONFIG_DIR = pathlib.Path(__file__).parent / "data" / "config"
```

Reply:
Thanks for the very good suggestion. I have opened a new PR for this.